ui/src/lib/gql/provider/GraphQLRequestMiddleware.ts

```
@@ -0,0 +1,200 @@
+  const form = new FormData();
+  const parsedBody = JSON.parse(request.body as string);
+  for (const file of files) {
+    //remove file here to reduce request size
+    set(parsedBody, file.variableKey[0], null);
+  }
+  console.log("parsed body: ", parsedBody);
+  form.append("operations", JSON.stringify(parsedBody));
+
+  const map = files.reduce((accumulator, { variableKey }, index) => {
+    return {
+      ...accumulator,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)